src/main/scala/edu/berkeley/ce/rockslicing/Delaunay.scala

```diff
@@ -62,7 +62,7 @@ object Delaunay {
     }
     */
     
-    val EPSILON = 0.0000001
+    val EPSILON = NumericUtils.EPSILON
```

Comment:
Instead of just changing the definition of `EPSILON` here, why don't we just change all references to `EPSILON` to references to `NumericUtils.EPSILON` in the code below?